packages/flutter_tools/lib/src/build_system/targets/windows.dart

```diff
@@ -187,42 +191,42 @@ class WindowsAotBundle extends Target {
 }
 
 class ReleaseBundleWindowsAssets extends BundleWindowsAssets {
-  const ReleaseBundleWindowsAssets();
+  const ReleaseBundleWindowsAssets(super.targetPlatform);
 
   @override
-  String get name => 'release_bundle_windows_assets';
+  String get name => 'release_bundle_${getNameForTargetPlatform(targetPlatform)}_assets';
```

Comment:
Potential concern: this will result in a name like `release_bundle_windows-x64_assets`. That's probably fine but we should double-check

Reply:
It's needed to avoid issue when compilation for both architectures.

Reply:
I'm more concerned by the `-` character, we'll want someone from the tool team to double-check this before merging :)

Reply:
For Linux, [it's done this way](https://github.com/flutter/flutter/blob/f5a983535131d948950124c53df6b8f491debd9f/packages/flutter_tools/lib/src/build_system/targets/linux.dart#L244), and presence of `-` seems to be fine.

Reply:
We can always lower this character into `_` if you think it's better.

Reply:
Ah okay, I'm fine with mirroring Linux here. Thanks for checking!